policyengine_us/variables/gov/states/mi/tax/income/deductions/retirement/tier_one/mi_retirement_benefits_deduction_tier_one.py

```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class mi_retirement_benefits_deduction_tier_one(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Michigan retirement benefits deduction for tier one"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "http://legislature.mi.gov/doc.aspx?mcl-206-30",  # (1)(f)
+        "https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/BOOK_MI-1040.pdf#page=17",
+        "https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits",
+    )
+    defined_for = "mi_retirement_benefits_deduction_tier_one_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mi.tax.income.deductions.retirement_benefits.tier_one
+        filing_status = tax_unit("filing_status", period)
+
+        person = tax_unit.members
+        uncapped_pension_income = person("taxable_pension_income", period)
```

Comment:
Unless this is limited to head_or_spouse, which the code doesn't currently reflect
```suggestion
        uncapped_pension_income = add(tax_unit, period, ["taxable_pension_income"])
```